core/res/res/values/cr_config.xml

```diff
@@ -19,6 +19,9 @@
     <!-- Whether device has dash charging support -->
     <bool name="config_hasDashCharger">false</bool>
 
+    <!-- Whether device has warp charging support -->
```

Comment:
XML entries should be in end. Not in middle.

Reply:
wanted to keep it near the one with Dash charge
doesn't matter that much anyway
want it on the end?

Reply:
okay.. can add. will add properly in next rebase.